governments.md

```diff
@@ -50,7 +50,7 @@
 
 [Shelling Out](https://nakamotoinstitute.org/shelling-out/)
 
-[The Bitcoin Standard (chapters 1-3)](https://saifedean.com/book/)
+[The Bitcoin Standard (chapters 1-3)](https://www.resistance.money/research/library/to%20be%20organised%20better/The%20Bitcoin%20Standard.pdf)
```

Comment:
Same here. No reason not to link to Saifedean's official page. We want to support Bitcoin content creators.